dataeng/resources/tableau-backup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -ex
+
+
+# This script ssh into Tableau server. Default centos user does not have access on Tableau directories
+# so it runs commands as root. Backups are being created using tsm maintenance backup utility. This script
+# copies backup file into s3 bucket and then remove backup file from tableau server.
+
+ssh -tt $USER_NAME@$TABLEAU_SERVER_IP -o StrictHostKeyChecking=no "S3_BUCKET=$S3_BUCKET" '
+
+# It gets the current date from system in 2021.10.05 format
+DATE="$(date '+%Y.%m.%d')"
+# It adds the date as prefix on backup name. Backup runs at 11:00 UTC so it also prefixed in the name.
+BACKUPNAME="$DATE.11.00-tableau-data-backup.tsbak"
+
+# Check if tableau backup file exist, exit if the file does not exist
+fileStatus=$([ -f /home/tableau_backups/$BACKUPNAME ] && echo 'Found' || echo 'NotFound' )
+if [[ "$fileStatus" == "NotFound" ]]
```

Comment:
Not a blocker, but you can actually just use the `find` command as the test in the `if` statement, like:
```
if [[ ! -f /home/tableau_backups/$BACKUPNAME ]] then
    echo "backup file not found"
    exit 1
fi
```